auth_cas/wizard/res_config.py

```diff
@@ -0,0 +1,134 @@
+                    # There is no error
+                    error = False
+                icp.set_param('cas_auth.cas_server', self.cas_server)
+                icp.set_param('cas_auth.cas_server_port',
+                              self.cas_server_port)
+            # Else, there is one error
+            # If the host field is valid, we save it and the default port value
+            elif self.cas_server:
+                icp.set_param('cas_auth.cas_server', self.cas_server)
+                icp.set_param('cas_auth.cas_server_port', default_port)
+            # Else, the host field is empty,
+            # but not the port field: we save it and the default host value
+            else:
+                icp.set_param('cas_auth.cas_server_port',
+                              self.cas_server_port)
+                icp.set_param('cas_auth.cas_server', default_host)
+        # If error is True, there is at least one error,
+        # so we deactivate CAS authentication
+        if error:
+            icp.set_param('cas_auth.cas_activated', 'False')
+            # If the host field is empty, we save his default value
+            if not self.cas_server:
+                icp.set_param('cas_auth.cas_server', default_host)
+            # Else, it seems it is the port field
+            # which is empty: we save his default value
+            else:
+                icp.set_param('cas_auth.cas_server_port', default_port)
+
+        # We save the field used to know
+        # if users have to be created on the fly or not
+        icp.set_param('cas_auth.cas_create_user', str(self.cas_create_user))
+
+    @api.multi
+    def check_cas_server(self):
+
+        """Check if CAS paramaters (host and port) are valids"""
+
+        self.ensure_one()
+        title = 'cas_check_fail'
+        message = 'Parameters are incorrect\nThere seems to be an ' \
+                  'error in the configuration.'
+        if self.cas_server_port == -1:
+            cas_server = self.cas_server
+        else:
+            url_server = urlparse(self.cas_server)
+            cas_server = \
+                url_server.scheme + '://' + url_server.netloc + \
+                ':' + str(self.cas_server_port) + url_server.path
+        try:
+            # The login function, from pycas,returns 3 if the host is
+            # a CAS host and if parameters given are bad
+            # (Parameters are empty, except the adress of the CAS server,
+            # so there are always bad !)
+            res = login(cas_server, ' ', ' ')
+            if res[0] == 3:
+                title = 'cas_check_success'
+                message = 'Parameters are correct\nThe CAS server is well ' \
+                          'configured !'
+        except Exception, e:
+            _logger.debug(e)
+        # At the moment, I only found this method
+        # in order to show a message after a request
+        raise ValidationError(_(title) + '\n' + _(message))
```

Comment:
I rather have a text field with the result of this check than raise a ValidationError 
Would be possible?